src/mmrelay/meshtastic_utils.py

```
@@ -740,12 +740,23 @@ def on_meshtastic_message(packet, interface):
+
     # Reaction handling (Meshtastic -> Matrix)
     # If replyId and emoji_flag are present and reactions are enabled, we relay as text reactions in Matrix
     if replyId and emoji_flag and interactions["reactions"]:
-        longname = get_longname(sender) or str(sender)
-        shortname = get_shortname(sender) or str(sender)
-        orig = get_message_map_by_meshtastic_id(replyId)
+        try:
+            longname = get_longname(sender) or str(sender)
+            shortname = get_shortname(sender) or str(sender)
+            orig = get_message_map_by_meshtastic_id(replyId)
+        except Exception:
```

Ruff (0.14.3)

Do not catch blind exception: `Exception`

(BLE001)